src/SFA.DAS.EmployerAccounts.Web/Extensions/AgreementTemplateExtensions.cs

```diff
@@ -0,0 +1,12 @@
+﻿using SFA.DAS.EmployerAccounts.Models.Account;
+
+namespace SFA.DAS.EmployerAccounts.Web.Extensions
+{
+    public static class AgreementTemplateExtensions
+    {
+        public static string GetInsetText(this AgreementTemplate agreementTemplate)
```

Comment:
I don't think I understand the value of moving this from a property to an extension method?